include/openenclave/tracee.h

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#ifndef _OE_TRACEE_H
+#define _OE_TRACEE_H
+
+#include <openenclave/bits/defs.h>
+#include <openenclave/enclave.h>
+#include <openenclave/log.h>
```

Comment:
Do you need to include log.h here?

Reply:
log.h is needed for "oe_log_level_t" used in the callback function pointer declaration